### ISSUE TITLE
Opening any "Linked Move" inside the .col files gives an error

### ISSUE THREAD

**Issue by Fexty12573** [Synthlight/MHW-Editor#11]
Opening any of the "Linked Move" tabs gives me this error:

![image](https://user-images.githubusercontent.com/60443001/85074560-0c35a900-b1bd-11ea-907c-de7f8acf17a0.png)

In Text:

```
---------------------------
Error Occured
---------------------------
Error occured. Press Ctrl+C to copy the contents of ths window and report to the developer.



System.InvalidOperationException: Dispatcher processing has been suspended, but messages are still being processed.
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.MessageBox(HandleRef hWnd, String text, String caption, Int32 type)
   at System.Windows.MessageBox.ShowCore(IntPtr owner, String messageBoxText, String caption, MessageBoxButton button, MessageBoxImage icon, MessageBoxResult defaultResult, MessageBoxOptions options)
   at MHW_Editor.Windows.MainWindow.ShowError(Exception err, String title) in R:\Games\Monster Hunter World\MHW-Editor\Windows\MainWindow.xaml.cs:line 717
   at MHW_Editor.Controls.MhwDataGridGeneric`1.On_AutoGeneratedColumns(Object sender, EventArgs e) in R:\Games\Monster Hunter World\MHW-Editor\Controls\MhwDataGrid.xaml.cs:line 294
   at System.Windows.Controls.DataGrid.OnAutoGeneratedColumns(EventArgs e)
   at System.Windows.Controls.DataGrid.AddAutoColumns()
   at System.Windows.Controls.DataGrid.MeasureOverride(Size availableSize)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at MS.Internal.Helper.MeasureElementWithSingleChild(UIElement element, Size constraint)
   at System.Windows.Controls.ContentPresenter.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Documents.AdornerDecorator.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Controls.Border.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Window.MeasureOverrideHelper(Size constraint)
   at System.Windows.Window.MeasureOverride(Size availableSize)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Interop.HwndSource.SetLayoutSize()
   at System.Windows.Interop.HwndSource.set_RootVisualInternal(Visual value)
   at System.Windows.Interop.HwndSource.set_RootVisual(Visual value)
   at System.Windows.Window.SetRootVisual()
   at System.Windows.Window.SetRootVisualAndUpdateSTC()
   at System.Windows.Window.SetupInitialState(Double requestedTop, Double requestedLeft, Double requestedWidth, Double requestedHeight)
   at System.Windows.Window.CreateSourceWindow(Boolean duringShow)
   at System.Windows.Window.CreateSourceWindowDuringShow()
   at System.Windows.Window.SafeCreateWindowDuringShow()
   at System.Windows.Window.ShowHelper(Object booleanBox)
   at System.Windows.Window.Show()
   at System.Windows.Window.ShowDialog()
   at MHW_Editor.Controls.MhwDataGridGeneric`1.On_Cell_Open_Click(Object sender, RoutedEventArgs e, PropertyInfo propertyInfo, String displayName) in R:\Games\Monster Hunter World\MHW-Editor\Controls\MhwDataGrid.xaml.cs:line 390
---------------------------
OK   
---------------------------
```

After that the program stops responding.

I've made sure that I'm using the files from the newest chunks so that can't be it.

>

**Comment by Synthlight** [comment]
This isn't your fault.
It's an oops I made with how the data grid handle filtering, and probably only for the specific case.
Fix is rather easy, just scroll down and open the full move list for now.
I've got some more stuff to add before I make a new release.

**Comment by Synthlight** [comment]
Fixed in v2.1.
